codeql_ruby/cwe-094/UnsafeCodeConstruction/impl/unsafeCode.rb

```
@@ -0,0 +1,122 @@
+    arr.push(x)
+# {fact rule=autoescape-disabled@v1.0 defects=1}
+
+    eval(arr.join("\n")) # NOT OK
+
+# {/fact}
+
+    arr2 = []
+    arr2 << y
+# {fact rule=autoescape-disabled@v1.0 defects=1}
+
+    eval(arr.join("\n")) # NOT OK
```

RuboCop (1.80.2)

[convention] The use of `eval` is a serious security risk.

(Security/Eval)